cmd/autocompletion.go

```diff
@@ -0,0 +1,242 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	autocompleteCmd.AddCommand(bashAutocompleteCmd)
+	autocompleteCmd.AddCommand(zshAutocompleteCmd)
+
+	RootCmd.AddCommand(autocompleteCmd)
+}
+
+var autocompleteCmd = &cobra.Command{
+	Use:   "completion",
+	Short: "Output shell completion code for the given shell (bash or zsh)",
+	Long: `
+Output shell completion code for bash or zsh
+This command prints shell code which must be evaluated to provide interactive
+completion of aws-okta commands.
+
+Bash
+	$ source <(aws-okta completion bash)
```

Comment:
I wonder if instead of subcommands, we could do a parameter defaulting to `auto` (the current shell).

- `bash -c 'aws-okta completion'` --> bash
- `bash -c `aws-okta completion bash` --> bash
- `zsh -c `aws-okta completion` --> zsh

Seems a little bit nicer, but not at all a blocker.

Reply:
I don't know how you can check this, any suggestion?

Reply:
I was thinking of just parsing `$SHELL`. I think that's pretty standard.

Reply:
$SHELL doesn't work if you change between shells, only for login shell. Maybe we can iterate it in another pr